### PR TITLE
Edu benefits copy 10-18

### PR DESCRIPTION
**Pull request by cweber**
This takes care of a handful of copy changes along with adding the required key to all pages, and beefing up the asterisk size. (#3257)

### CODE REVIEW

File: src/js/edu-benefits/components/military-history/MilitaryServiceReview.jsx [hunk 1]
```diff
@@ -45,20 +48,12 @@ export default class MilitaryServiceReview extends React.Component {
...
-              </tr>
-              <tr>
-                <td>Apply this service period to the benefit I'm applying for</td>
+                <td>Apply this service period to the benefit I’m applying for.</td>
                 <td>{tour.applyPeriodToSelected ? 'Yes' : 'No'}</td>
               </tr>
               {!tour.applyPeriodToSelected
                 ? <tr>
-                  <td>Please explain how you'd like this service period applied.</td>
+                  <td>Please explain how you’d like this service period applied.</td>
```
Comment (jbalboni) [inline src/js/edu-benefits/components/military-history/MilitaryServiceReview.jsx]: I usually replace these quotes with plain `'`.

File: src/js/edu-benefits/components/personal-information/DependentInformationReview.jsx [hunk 2]
```diff
@@ -11,11 +11,11 @@ export default class DependentInformationReview extends React.Component {
             <td>{showYesNo(this.props.data.serviceBefore1977.married)}</td>
           </tr>
           <tr>
-            <td>Do you have any children who are under age 18, or over age 18 but under 23, not married and attending school, or of any age permanently disabled for mental or physical reasons?</td>
+            <td>Do you have any children who are under age 18? Or do you have any children who are over age 18 but under 23, not married, and attending school? Or do you have any children of any age who are permanently disabled for mental or physical reasons?</td>
```
Comment (jbalboni) [inline src/js/edu-benefits/components/personal-information/DependentInformationReview.jsx]: Do we need to make this update for the review page, too?

File: src/js/edu-benefits/components/school-selection/SchoolSelectionReview.jsx [hunk 3]
```diff
@@ -45,12 +45,12 @@ export default class SchoolSelectionReview extends React.Component {
...
           </tr>
           <tr>
-            <td>The date your training began or will begin.</td>
+            <td>The date your training began or will begin:</td>
             <td>{this.props.data.educationStartDate.month.value ? `${this.props.data.educationStartDate.month.value}/${this.props.data.educationStartDate.day.value}/${this.props.data.educationStartDate.year.value}` : null}</td>
           </tr>
           {this.props.data.currentlyActiveDuty.yes.value === 'Y'
             ? <tr>
-              <td>Are you receiving, or do you expect to receive any money (including, but not limited to, federal tuition assistance) from the armed forces or public health services for any part of your coursework or training?</td>
+              <td>Are you getting, or do you expect to get any money (including, but not limited to, federal tuition assistance) from the Armed Forces or public health services for any part of your coursework or training?</td>
```
Comment (jbalboni) [inline src/js/edu-benefits/components/school-selection/SchoolSelectionReview.jsx]: Should we make this change on the form field label, too?

**Review by jbalboni — APPROVED** [review]
This looks fine to me. I had a couple questions that are mostly around making sure changes are made for both the form field labels and the labels on the review page.

File: src/js/edu-benefits/components/military-history/MilitaryServiceReview.jsx (hunk 1 above)
Comment (goldenmeanie) [inline src/js/edu-benefits/components/military-history/MilitaryServiceReview.jsx]: Is that for a technical reason? Would prefer real apostrophes unless it is going to break something.

File: src/js/edu-benefits/components/personal-information/DependentInformationReview.jsx (hunk 2 above)
Comment (goldenmeanie) [inline src/js/edu-benefits/components/personal-information/DependentInformationReview.jsx]: I'll double check, but yes I think in almost every case, they should match.

File: src/js/edu-benefits/components/school-selection/SchoolSelectionReview.jsx (hunk 3 above)
Comment (goldenmeanie) [inline src/js/edu-benefits/components/school-selection/SchoolSelectionReview.jsx]: Yes, this should match.

File: src/js/edu-benefits/components/military-history/MilitaryServiceReview.jsx (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/components/military-history/MilitaryServiceReview.jsx]: If it looks ok in the browser, it should be fine. It's just easy to have character encoding issues.

**Comment by cweber** [comment]
Wooo!!!!

**Pull request merged**